libraries/monitor/src/monitor.js

```
@@ -181,18 +181,24 @@ class Monitor {
    */
   async oneShot(name, fn) {
     let exitStatus = 0;
-
+    let error = undefined;
+    const start = process.hrtime();
     try {
-      try {
-        assert.equal(typeof name, 'string');
-        assert.equal(typeof fn, 'function');
+      assert.equal(typeof name, 'string');
+      assert.equal(typeof fn, 'function');
 
-        await this.timer(name, fn);
-      } catch (err) {
-        this.reportError(err);
```

Same question here: why not report these errors?